transformer_engine/jax/cpp_extensions/activation.py

```diff
@@ -155,6 +189,9 @@ def act_lu(inputs: jnp.ndarray, activation_type: Sequence[Union[str, Callable]])
     Input shape: (N, 1, H) for non-gated activations
                  (N, 2, H) for gated activations
     """
+    if not ActLuPrimitive.enabled():
+        return _act_lu(inputs, activation_type)
```

Comment:
I think it is better to rename this function to `jax_act_lu()`

Reply:
Named as `_jax_act_lu` and also other functions.